config/importmap.rb

```diff
@@ -5,3 +5,4 @@
 pin "@hotwired/stimulus", to: "stimulus.min.js", preload: true
 pin "@hotwired/stimulus-loading", to: "stimulus-loading.js", preload: true
 pin_all_from "app/javascript/controllers", under: "controllers"
+# pin_all_from "app/javascript/custom", under: "custom"
```

Comment:
Code that you do not use, do not comment on it, delete it.

Reply:
Perfect, we will eliminate the unnecessary code.